templates/client-daemonset.yaml

```diff
@@ -85,6 +93,18 @@ spec:
                 -advertise="${POD_IP}" \
                 -bind=0.0.0.0 \
                 -client=0.0.0.0 \
+                {{- if .Values.global.tls.enabled }}
+                -hcl='ca_file = "/consul/tls/ca/tls.crt"' \
+                -hcl='cert_file = "/consul/tls/client/tls.crt"' \
+                -hcl='key_file = "/consul/tls/client/tls.key"' \
+                -hcl='verify_incoming = {{ .Values.global.tls.verifyIncoming }}' \
```

Comment:
There is also `verify_incoming_https` and `verify_incoming_rpc` which is useful if you want to use the UI via HTTPS.

Reply:
I think I like that approach better. Less pain to go through with HTTPS clients.

Reply:
Fixed with dd7a23536bc179d0f0b8a646c94bea291d3e3c00